.idea/copyright/profiles_settings.xml

```diff
@@ -0,0 +1,7 @@
+<component name="CopyrightManager">
+  <settings>
+    <module2copyright>
+      <element module="All" copyright="GPLv3" />
+    </module2copyright>
+  </settings>
+</component>
```

Comment:
This line doesn't end with a newline character.

Reply:
Intelij gets mad if you add a newline in this